clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -335,4 +336,24 @@ private void updateBlockInStream(long blockId) throws IOException {
       mShouldCacheCurrentBlock = mTachyonStorageType.isStore();
     }
   }
+
+  class PrivateAccess {
+    public boolean isClosed() {
+      return mClosed;
+    }
+
+    public boolean shouldCacheCurrentBlock() {
+      return mShouldCacheCurrentBlock;
+    }
+
+    public void setCurrentInstream(BlockInStream inStream) {
+      mCurrentBlockInStream = inStream;
+    }
+  }
+
+  /** Grants access to private members to testers of this class. */
+  @Override
+  public void grantAccess(Tester<FileInStream> tester) {
+    tester.receiveAccess(new PrivateAccess());
+  }
```

Comment:
Is it possible to remove the `PrivateAccess` and just use Powermock's `getInternalState`?